### PR TITLE
Create new-file.txt

### PR DESCRIPTION
**Pull request by jmdc2071**
New feature4 implemented

### CODE REVIEW

**Review by jmdc2071 — COMMENTED** [review]
Thanks for another great feature

File: feature4/new-file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Some text for the new awsome feature
```
Comment (jmdc2071) [inline feature4/new-file.txt:1]: looks good

**Review by jmdc2071 — COMMENTED** [review]
done

**Review by jeffbxb — APPROVED** [review]
done

**Pull request merged**